querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/MetaDataSerializer.java

```diff
@@ -110,6 +111,13 @@ protected void constructorsForVariables(CodeWriter writer, EntityType model) thr
                 + "forVariable(variable), schema, table);");
         constructorContent(writer, model);
         writer.end();
+
+        writer.beginConstructor(new Parameter("variable", Types.STRING),
+                                new Parameter("schema", Types.STRING));
+        writer.line(SUPER, "(", writer.getClassConstant(localName) + COMMA
+                + "forVariable(variable), schema, \"", model.getData().get("table") + "\");");
```

Comment:
Take it please.

Thank you 

Sent from my iPhone

> On 12/08/2016, at 12:24, Ruben Dijkstra <notifications@github.com> wrote:
> 
> In querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/MetaDataSerializer.java:
> 
> > @@ -110,6 +111,13 @@ protected void constructorsForVariables(CodeWriter writer, EntityType model) thr
> >                  + "forVariable(variable), schema, table);");
> >          constructorContent(writer, model);
> >          writer.end();
> > +
> > +        writer.beginConstructor(new Parameter("variable", Types.STRING),
> > +                                new Parameter("schema", Types.STRING));
> > +        writer.line(SUPER, "(", writer.getClassConstant(localName) + COMMA
> > +                + "forVariable(variable), schema, \"", model.getData().get("table") + "\");");
> That's fine, do you want us to wait, or shall we take it from here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>